code/modules/mob/living/carbon/human/species_types/golems.dm

```diff
@@ -312,27 +312,33 @@
 		H.reagents.remove_reagent(chem.type, REAGENTS_METABOLISM)
 		return 1
 
-//Radioactive
+//Radioactive puncher, hits for burn but only as hard as human, slightly more durable against brute but less against everything else
 /datum/species/golem/uranium
 	name = "Uranium Golem"
 	id = "uranium golem"
 	fixed_mut_color = "7f0"
 	meat = /obj/item/stack/ore/uranium
-	info_text = "As an <span class='danger'>Uranium Golem</span>, you emit radiation pulses every once in a while. It won't harm fellow golems, but organic lifeforms will be affected."
-
+	info_text = "As an <span class='danger'>Uranium Golem</span>, your very touch burns and irradiates organic lifeforms. You don't hit as hard as most golems, but you are far more durable against blunt force trauma."
+	attack_verb = "burn"
+	attack_sound = 'sound/weapons/sear.ogg'
+	attack_type = BURN
 	var/last_event = 0
 	var/active = null
+	armor = 40
+	brutemod = 0.5
+	punchdamagelow = 1
+	punchdamagehigh = 10
+	punchstunthreshold = 9
```

Comment:
humans have a punchstunthreshold of 10, actually

Reply:
Well...I guess they get a slightly, slightly higher chance at a crit punch.